techniques/system/inventory/1.0/fusionAgent.st

```diff
@@ -772,6 +772,12 @@ bundle agent sendInventory
       "any" usebundle => rudder_common_report("Inventory", "log_warn", "&TRACKINGKEY&", "inventory", "None", "Could not delete inventory file after sending to server");
 
 }
+body file_select by_name_old(names,days)
```

Comment:
could you rename it (for example with a fusion prefix)) to allow easy move to ncf?